src/components/Dashboard/Content/index.tsx

```diff
@@ -26,6 +28,18 @@ const DashboardContent = ({
   onNextSurvey,
   onIndicatorTapped,
 }: DashboardContentProps): JSX.Element => {
+  const getCurrentSurvey = (): Survey | undefined => {
+    return _get(surveys, currentPosition, undefined);
+  };
+
+  const getSurveyPath = (): string => {
+    const currentSurvey = getCurrentSurvey();
```

Comment:
should we move it out, I see we use the `getCurrentSurvey()` quite often

Reply:
@nvminhtue fixed in [29fe872](https://github.com/manh-t/react-survey/pull/61/commits/29fe8721eeffdb78f076407cab85b9c967a5f030)